site-new/src/components/animated-logo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import gifUrl from '@site/static/img/armeria.gif';
+import videoUrl from '@site/static/img/armeria.m4v';
```

Comment:
It would be nice to have the dark version of the animated logo. I have no idea how to create one, though :sweat_smile:

Reply:
Let’s keep using it as is for now. From what I recall, it was outsourced to a professional agency.